### PR TITLE
Login, sign up, reset password CSS

### PR DESCRIPTION
**Pull request by rdorame**
CSS to Login, Password reset, Sign up

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -44,7 +45,7 @@ h1,h2,h3,h4,h5,h6 {
 }
 
 .jumbotron {
-    background: linear-gradient(rgba(0,0,0,0.5), rgba(0,0,0,0.5)), url('../img/action.jpeg'); 
+    background: linear-gradient(rgba(0,0,0,0.5), rgba(0,0,0,0.5)), url('../img/action.jpeg');
```
Comment (rikyfocil-old) [inline public/css/style.css:49]: Where does ../img/action.jpeg suppose to be?

File: public/css/style.css [hunk 2]
```diff
@@ -95,7 +96,7 @@ h1,h2,h3,h4,h5,h6 {
     -o-background-size: 100%;
     background-size: 100%;
     /* background: url('../img/sun.jpeg') 50% 50% no-repeat; */
-    background: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.3)), url('../img/sun.jpeg') 50% 30%; 
+    background: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.3)), url('../img/sun.jpeg') 50% 30%;
```
Comment (rikyfocil-old) [inline public/css/style.css:100]: Same as above

File: resources/views/auth/login.blade.php [hunk 3]
```diff
@@ -1,68 +1,62 @@
...
-                <div class="panel-heading">Login</div>
-                <div class="panel-body">
-                    <form class="form-horizontal" role="form" method="POST" action="{{ url('/login') }}">
-                        {{ csrf_field() }}
+  <div class="text-center" style="padding:50px 0">
+
+  <div class = "login-form-1">
+    <h1> Login </h1>
+
+    <form id="login-form" class="text-left" role="form" method="POST" action="{{ url('/login') }}">
```
Comment (rikyfocil-old) [inline resources/views/auth/login.blade.php]: Its better to use route()

File: resources/views/auth/register.blade.php [hunk 4]
```diff
@@ -1,82 +1,82 @@
...
+
+        <div class="form-group login-group-checkbox">
+          <input type="checkbox" class="" id="reg_agree" name="reg_agree">
+          <label for="reg_agree">I agree with <a href="#">terms & conditions</a></label>
+        </div>
+      </div>
+      <button type="submit" class="login-button"> <p>&#10217;</p> </button>
+    </div>
+    <div class="etc-login-form">
+      <p><a href="#">Already have an account?</a></p>
```
Comment (rikyfocil-old) [inline resources/views/auth/register.blade.php]: Change url to login

**Review by rikyfocil-old — CHANGES_REQUESTED** [review]

File: resources/views/auth/register.blade.php [hunk 5]
```diff
@@ -1,82 +1,82 @@
...
+                @if ($errors->has('password_confirmation'))
+                    <span class="help-block">
+                        <strong>{{ $errors->first('password_confirmation') }}</strong>
+                    </span>
+                @endif
         </div>
+
+        <div class="form-group login-group-checkbox">
+          <input type="checkbox" class="" id="reg_agree" name="reg_agree">
+          <label for="reg_agree">I agree with <a href="#">terms & conditions</a></label>
```
Comment (Raul-MA) [inline resources/views/auth/register.blade.php]: Unsupported. Please change controller or remove checkbox.

**Review by Raul-MA — DISMISSED** [review]

**Review by rdorame — COMMENTED** [review]
Done

File: resources/views/auth/passwords/email.blade.php [hunk 6]
```diff
@@ -1,47 +1,43 @@
...
-                        {{ csrf_field() }}
+  <div class="text-center" style="padding:50px 0">
+  <h1> Reset password </h1>
+    @if (session('status'))
+        <div class="alert alert-success">
+            {{ session('status') }}
+        </div>
+    @endif
+  	<div class="login-form-1">
+  		<form class="text-left" role="form" method="POST" action="{{ url('/password/email') }}">
```
Comment (rikyfocil-old) [inline resources/views/auth/passwords/email.blade.php]: action with route please

**Review by rikyfocil-old — CHANGES_REQUESTED** [review]

**Review by rikyfocil-old — APPROVED** [review]

**Pull request merged**